scripts/build.py

```diff
@@ -66,22 +72,51 @@ def checks(self):
         build_path = self._get_swift_build_path()
         os.chdir(build_path)
         if self._should_run_checks():
-            subprocess.check_call([self.make_cmd, 'check', '-j1'], env=dict(os.environ))
+            subprocess.check_call(["cmake", "--build", ".", "--target", "test"], env=dict(os.environ))
```

Comment:
How about `subprocess.check_call(["ctest", "--verbose"]` so we see the full test output on CI